ps/jenkins/package-testing-ps-build-8.1.groovy

```diff
@@ -0,0 +1,273 @@
+library changelog: false, identifier: 'lib@PS-8.1-test', retriever: modernSCM([
+    $class: 'GitSCMSource',
+    remote: 'https://github.com/kaushikpuneet07/jenkins-pipelines.git'
+]) _
+
+setup_rhel_package_tests = { ->
+    sh '''
+        sudo yum install -y epel-release
+        sudo yum -y update
+        sudo yum install -y ansible git wget tar
+    '''
+}
+
+setup_rhel8_package_tests = { ->
+    sh '''
+        sudo yum install -y epel-release
+        sudo yum -y update
+        sudo yum install -y ansible-2.9.27 git wget tar
+    '''
+}
+
+
+setup_debian_package_tests = { ->
+    sh '''
+        sudo apt-get update
+        sudo apt-get install -y ansible git wget
+    '''
+}
+
+setup_ubuntu_package_tests = { ->
+    sh '''
+        sudo apt-get update
+        sudo apt-get install -y software-properties-common
+        sudo apt-add-repository --yes --update ppa:ansible/ansible
+        sudo apt-get install -y ansible git wget
+    '''
+}
+
+node_setups = [
+    "min-buster-x64": setup_debian_package_tests,
+    "min-bullseye-x64": setup_debian_package_tests,
+    "min-bookworm-x64": setup_debian_package_tests,
+    "min-ol-8-x64": setup_rhel8_package_tests,
+    "min-centos-7-x64": setup_rhel_package_tests,
+    "min-focal-x64": setup_ubuntu_package_tests,
+    "min-amazon-2-x64": setup_rhel_package_tests,
+    "min-jammy-x64": setup_ubuntu_package_tests,
+    "min-ol-9-x64": setup_rhel_package_tests,
+]
+
+void setup_package_tests() {
+    node_setups[params.node_to_test]()
+}
+
+List all_nodes = node_setups.keySet().collect()
+
+List all_actions = [
+    "install",
+    "upgrade",
+    "maj-upgrade-to",
+    "kmip",
+    "kms",
+]
+
+product_action_playbooks = [
+    ps81: [
+        install: "ps_81.yml",
```

Comment:
The same here: are we going to create/update playbooks on every innovations release?